roles/hue/server/templates/hue.ini.j2

```diff
@@ -1299,18 +1299,20 @@ ssl_cert_ca_verify={{ hadoop.yarn.ssl_cert_ca_verify }}
   # Redefine different properties there.
   # e.g.
 
-[[[ha]]]
-resourcemanager_host={{ hadoop.yarn.ha.resourcemanager_host }}
-resourcemanager_port={{ hadoop.yarn.ha.resourcemanager_port }}
-submit_to={{ hadoop.yarn.ha.submit_to }}
-logical_name={{ hadoop.yarn.ha.logical_name }}
-security_enabled={{ hadoop.yarn.ha.security_enabled }}
-resourcemanager_api_url={{ hadoop.yarn.ha.resourcemanager_api_url }}
-history_server_api_url={{ hadoop.yarn.ha.history_server_api_url }}
-spark_history_server_url={{ hadoop.yarn.ha.spark_history_server_url }}
-spark_history_server_security_enabled={{ hadoop.yarn.ha.spark_history_server_security_enabled }}
-ssl_cert_ca_verify={{ hadoop.yarn.ha.ssl_cert_ca_verify }}
+{% for i in range(1, groups['yarn_rm'] | length) %}
+{{ '[[[ha' + i | string + ']]]' }}
+resourcemanager_host={{ hadoop.yarn.resourcemanager_hosts[i] }}
```

Comment:
`i` start at `1`, are you sure for `hadoop.yarn.resourcemanager_hosts[i]` ? The first host is at `0` no ? Or you want only the second and omit the first ?